src/pds/ingress/pds-nucleus-datasync-completion-trigger.py

```diff
@@ -0,0 +1,70 @@
+"""
+============================================================
+pds-nucleus-datasync-completion-trigger.py
+============================================================    =
+
+Lambda function that get triggered when the PDS Nucleus Staging S3 Bucket to EFS datasync task is completed.
+This function reads the list of data transfer verification reports created by a specific DataSync task
+and asynchronously call the pds-nucleus-datasync-completion.py for each data transfer verification
+report JSON file.
+
+"""
+
+import boto3
+import logging
+import json
+import os
+
+datasync_reports_s3_bucket_name = "pds-nucleus-datassync-reports"
+
+logger = logging.getLogger("pds-nucleus-datasync-completion-trigger")
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler())
+
+s3 = boto3.resource('s3')
+s3_client = boto3.client('s3')
+
+s3_bucket_name = "pds-nucleus-staging"
+db_clust_arn = os.environ.get('DB_CLUSTER_ARN')
+db_secret_arn = os.environ.get('DB_SECRET_ARN')
+efs_mount_path = os.environ.get('EFS_MOUNT_PATH')
+
+rds_data = boto3.client('rds-data')
+
+# Define the client to interact with AWS Lambda
+client = boto3.client('lambda')
+
+
+def lambda_handler(event, context):
+    """ Lambda Handler """
```

Comment:
Hahah maybe try a little harder? 😄 A comment that just restates the function name doesn't add that much 😉

Reply:
`lambda_handler` is the default endpoint of lambda and I thought it is anyway self explanatory 😄 . I updated the code with the new comment `""" Lambda Handler - The entry point of lambda """ `